src/www/head.inc

```diff
@@ -42,6 +42,13 @@ $pagetitle .= html_safe(sprintf(' | %s.%s', $config['system']['hostname'], $conf
     <meta name="copyright" content="" />
     <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1" />
 
+    <script>
+      // Prevent Chrome from disabling double tap zoom based on viewport width <= device-width.
+      var device_width = screen.width + 1;
+      var viewport = document.querySelector("meta[name=viewport]");
+      viewport.setAttribute('content', 'width='+ device_width +', initial-scale=1, minimum-scale=1');
+    </script>
+
```

Comment:
use the partial form authgui.inc when created

Reply:
Can you point me at an example I can follow?
Thanks

Reply:
What I am thinking about is having a function to generate the head stuff that is identical at every place. In that case you just need to have your change at that place and a second time for MVC.

probably a function like

```php
<?php
function create_head_base_content($themename) {
?>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <meta name="robots" content="noindex, nofollow, noodp, noydir" />
    <meta name="keywords" content="" />
    <meta name="description" content="" />
    <meta name="copyright" content="" />
    <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1" />

    <script>
      // Prevent Chrome from disabling double tap zoom based on viewport width <= device-width.
      var device_width = screen.width + 1;
      var viewport = document.querySelector("meta[name=viewport]");
      viewport.setAttribute('content', 'width='+ device_width +', initial-scale=1, minimum-scale=1');
    </script>

    <link href="/ui/themes/<?= $themename ?>/build/css/main.css" rel="stylesheet">
    <link href="/ui/themes/<?= $themename ?>/build/images/favicon.png" rel="shortcut icon">

    <script src="/ui/js/jquery-3.2.1.min.js"></script>
    <script src="/ui/js/jquery-migrate-3.0.1.min.js"></script>

<?php
}
?>
```

So the other pages don't need to include have the same headers - you can simply do something like this:
```php
<?php create_head_base_content($themename); ?>
```

Reply:
It is just for not repeating any stuff on any page.  The block does not include the title tag or something page specific. Maybe even more JS libraries can be moved inside.

Reply:
Have this working on the legacy side.  Don't know how to deal with the PHP function in MVC side.  Where to put the PHP and how to properly include/call it from in the .volt files.  Need some guidance.

On the legacy side I have the PHP function in authgui.inc

Reply:
MVC supports partials: by default but as far as I know there is no need for that because there is only a single layout.

Here is an example for how to use a partial:
https://github.com/opnsense/core/blob/master/src/opnsense/mvc/app/views/layout_partials/base_form.volt#L104

Reply:
There is the MVC default and captive portal has it's own head as well.  Maybe captive portal needs to be reworked to use the default.